index.html

```diff
@@ -0,0 +1,633 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tezo</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./css/style.css">
+</head>
+<body>
+    <!--searchbar and sidebar -->
+    <div class="header-ul">
+        <div onclick=showsidebar() class="menu">
+            <svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 -960 960 960" width="24"><path d="M120-240v-80h720v80H120Zm0-200v-80h720v80H120Zm0-200v-80h720v80H120Z"/></svg>
+        </div>
+        <div>
+            <div class="header-searchbar">
+                <div class="searchpart">
+                    <img src="./images/bx-search.png" alt="search symbol">
+                    <input type="search" name="search"  id="search" placeholder="Search (Ctrl+/)">
```

Comment:
Search box input size is too small. It should have same size. It should occupy entire width until icons.